evap/staff/views.py

```diff
@@ -220,7 +220,18 @@ def apply(request, evaluations, email_template=None, email_template_contributor=
         messages.success(request, ungettext("Successfully enabled {} evaluation for editor review.",
             "Successfully enabled {} evaluations for editor review.", len(evaluations)).format(len(evaluations)))
         if email_template:
-            email_template.send_to_users_in_evaluations(evaluations, [EmailTemplate.EDITORS], use_cc=True, request=request)
+            evaluations_by_responsible = {}
```

Comment:
have you considered moving that logic into send_to_users_in_evaluations?

Reply:
Yes, I considered it, but I currently do not see a great benefit here as this case currentlyexists only once.
If we want to keep the abstraction level of `send_to_users_in_evaluations`, a new parameter would be necessary, like `cc_groups` which controls which groups should be CC’ed.
For editors, we decided for some trade-off here as some editors may receive mails whereas only one displayed evaluation is relevant to them. It may be fine in this particular case, but it should be reconsidered for other groups.
Of course, we don’t have to choose this rather abstract level, but then we have a parameter which is mostly unused and makes `send_to_users_in_evaluations` more verbose.
Or do you have another option in your mind?